CHANGELOG.md

```diff
@@ -1,4 +1,6 @@
-# Upcoming version
+# Upcoming
```

Comment:
Is there a reason for rename?

Reply:
Just a miss on my part - I renamed the old heading to `v0.4.0` and then re-typed the "Upcoming" thing

Reply:
Fixed (and also changed the overall formatting to match what the other rust-vmm crates use here)